{"name": "Проверка получения ошибки \"Учетная запись не найдена\" при передачи в теле запроса неправильных данных полей логина или пароля", "status": "passed", "parameters": [{"name": "incorrectly_filled_in_fields", "value": "{'login': 'mejiaju', 'password': 'invalid_password'}"}], "start": 1744212962603, "stop": 1744212962815, "uuid": "9fcd252c-b28e-4dad-ac51-7322cc6d8568", "historyId": "3ddda1ebf9f0b1145b327434736b6030", "testCaseId": "a6fe86b6256607a65c10d1f15a41e383", "fullName": "tests.test_check_login_courier.TestCheckLoginCourier#test_login_courier_with_incorrect_loginor_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_check_login_courier"}, {"name": "subSuite", "value": "TestCheckLoginCourier"}, {"name": "host", "value": "Martin"}, {"name": "thread", "value": "15620-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_check_login_courier"}]}